libimage/save.go

```diff
@@ -56,13 +56,13 @@ func (r *Runtime) Save(ctx context.Context, names []string, format, path string,
 
 	// Dispatch the save operations.
 	switch format {
-	case "oci-archive", "oci-dir", "docker-dir":
+	case "oci-archive", "oci-dir", "docker-dir", "dir":
```

Comment:
Why `dir`?  That is not supported by `podman save`.

Reply:
Could we then make `OCI` the default for `dir`?  Currently, it's `docker`.

Reply:
That is a change in transports though?  Would you want this to conflict?

Reply:
The c/image `dir:` transport is “raw unmodified accept-anything”, it’s not tied to a manifest format like Docker/OCI, and it’s specified by neither.

I completely support not naming that destination `docker-dir`.

Making ”OCI” (the manifest format?) default for `dir:` (the transport) would defeat the original “debugging destination” purpose in c/image; I suppose Podman could set up saves to use the `dir:` format and force a conversion, but that seems worse in every way than just using the `oci:` (oci/layout) transport and including an OCI-native index blob and the like.

---

WRT whether `podman save` should default to `dir:`, `oci:` (a directory), or `docker-archive:` / `oci-archive:`, I don’t have a strong preference (I’d weakly say that the `dir:` debugging format is probably not the right default) — anyway that seems like a topic for a different PR.

Reply:
I am not interested in changing the default.  Just in removing the "docker" adjective from our man pages when possible, especially when it adds little value. 10 instances of Docker in the podman save man page, none of which add any value to the user IMHO.

```
 man  podman save | grep docker
       podman  save saves an image to either docker-archive, oci-archive, oci-
       dir (directory with oci manifest type), or docker-dir  (directory  with
       v2s2  manifest  type)  on the local machine, default is docker-archive.
       dir transport i.e --format=oci-dir or --format=docker-dir
       Save  image  to  docker-archive,  oci-archive  (see   containers-trans‐
       ports(5)),  oci-dir  (oci transport), or docker-dir (dir transport with
              --format docker-archive
              --format docker-dir
       docker-archive.  The default for this option can be  modified  via  the
              $ podman save --format docker-dir -o ubuntu-dir ubuntu
```

Reply:
If it’s just that, the man page lists the available options three times, when doing so just once in the `--format` section would probably suffice.

The default value, and the conditionals for `--compress` and `--multi-image-archive`, can’t be eliminated that easily, to be fair.

Reply:
> the man page lists the available options three times, when doing so just once in the `--format` section would probably suffice.

https://github.com/containers/podman/pull/12489